lib/actions/preload.coffee

```diff
@@ -190,12 +212,12 @@ module.exports =
 		options.splashImage = options['splash-image']
 		delete options['splash-image']
 
-		options.dontCheckDeviceType = options['dont-check-device-type']
-		delete options['dont-check-device-type']
-		if options.dontCheckDeviceType and not options.appId
+		options.dontCheckArch = options['dont-check-arch'] || false
+		delete options['dont-check-arch']
+		if options.dontCheckArch and not options.appId
 			exitWithExpectedError('You need to specify an app id if you disable the device type check.')
```

Comment:
NIT: should this be `if you disable the device architecture type check`?

Reply:
good catch, updated